packages/@lwc/babel-plugin-component/src/__tests__/fixtures/component/camel-to-kebab-component-name/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "namespace": "x",
+    "name": "camelCase"
```

Comment:
The [rollup plugin doesn't format the names](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/rollup-plugin/src/index.ts#L302-L303) and I also verified that LCS provides the names in camel casing as well.

As a side note, the component name used by the `stylesheetToken` is not kebab cased, not sure if this is something we want to convert but it may be a breaking change.